Emulator/PCMCIA/TPCMCIAController.h

```diff
@@ -299,7 +299,7 @@ class TPCMCIAController
 										///< R/W for GetChipInfo.
 	KUInt32				mReg_3C00;		///< Value of xC003C00 (w   - ?? )
 	KUInt32				mReg_4000;		///< Value of xC004000 (w   - ?? )
-	KUInt32				mReg_4400;		//                     (    - ?? )
+//	KUInt32				mReg_4400;		//                     (    - ?? )
```

Comment:
I would drop the line or fix the comment: the register is only read and always reads 0xFC.
Not sure what it means, though.